pkg/model/postgresql_deployment.go

```diff
@@ -142,6 +142,8 @@ func PostgresqlDeployment(cr *v1alpha1.Keycloak, isOpenshift bool) *v13.Deployme
 							},
 						},
 					},
+					Tolerations:  cr.Spec.KeycloakDeploymentSpec.Tolerations,
+					NodeSelector: cr.Spec.KeycloakDeploymentSpec.NodeSelector,
```

Comment:
The embedded DB is [deprecated](https://www.keycloak.org/docs/latest/release_notes/index.html#deprecated-features-in-the-keycloak-operator). We should not do any changes to it. Could you please remove it?